modules/swagger-codegen/src/main/resources/typescript-fetch/modelEnum.mustache

```diff
@@ -0,0 +1,12 @@
+/**
+ * {{{description}}}
+ * @export
+ * @enum {string}
+ */
+export enum {{classname}} {
+{{#allowableValues}}
+{{#enumVars}}
+    {{{name}}} = <any> {{{value}}}{{^-last}},{{/-last}}
+{{/enumVars}}
+{{/allowableValues}}
+}
```

Comment:
The file doesn't end with a newline symbol. Take a look at this link what this means https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline.